tsconfig.json

```diff
@@ -4,8 +4,8 @@
     ],
```

Comment:
...  but you should add config.ts to your files array! That's why it keep giving you a red squiggly for it.

Reply:
done